torchbenchmark/util/backends/torchdynamo.py

```diff
@@ -0,0 +1,49 @@
+"""
+Support TorchDynamo(https://github.com/facebookresearch/torchdynamo) backends
+"""
+import argparse
+from typing import List
+
+class NullContext:
+    def __enter__(self):
+        pass
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        pass
+
+def help(fn):
+    return fn.__doc__
```

Comment:
This likely isn't needed.  Just put the docs in `help="blah blah"`

Reply:
Removed.